### DISCUSSION TITLE
use graphar-spark library， run the examples occurs error：ArrayIndexOutOfBoundsException: 34826

### DISCUSSION THREAD

**Discussion by DancyHuang** [apache/incubator-graphar#598] [Q&A] [answered]
use Vertex_reader read the parquet/ orc/ csv files of graph ldbc_sample, occurs the errors below:
![IMG_20240819_171001](https://github.com/user-attachments/assets/84cea069-f0b9-429e-a34e-664ad6cda828)

**Comment by SemyonSinchenko** [comment]
Hi @DancyHuang! Thanks for sharing! May you also share the version of GraphAr Spark and a version of Apache Spark you are using?

**Reply by DancyHuang** [reply]
The version of GraphAr is 0.12.0  and the version of Apache Spark is 3.2.2

**Comment by acezen** [comment]
Hi, @DancyHuang , can you post the full log of the error, and the vertex yaml you are using?  from the picture, it looks like the yaml can not be parsed with graphar spark.

**Reply by DancyHuang** [reply]
the full error log：
![IMG_20240820_103407](https://github.com/user-attachments/assets/6a1eeab1-740a-45a2-9ae4-2b6c9bcd2bba)
![IMG_20240820_103203](https://github.com/user-attachments/assets/60cabd01-c7fc-49eb-b9fe-c5c135a0ce3e)

and the vertex yaml is https://github.com/apache/incubator-graphar-testing/blob/main/ldbc_sample/parquet/person.vertex.yml

![IMG_20240820_103334](https://github.com/user-attachments/assets/8d5af7eb-4362-4fb5-9f01-a9ba1cf4c85f)

**Reply by DancyHuang** [reply, marked as answer]
The example runs ok now.  It might be due to a conflict with the JAR packages I've included.
